### DISCUSSION TITLE
Can dex be connected to aurora server less Postgres (10.12)

### DISCUSSION THREAD

**Discussion by AnkitaBarbi** [dexidp/dex#1985] [Q&A] [answered]
Can dex be connected to aurora server less Postgres (10.12).  Is this version supported?

**Comment by heidemn** [comment, marked as answer]
@AnkitaBarbi probably you should ask your question in this repo: https://github.com/lib/pq 
That's the Go Postgres driver which is used in `storage/sql/sql.go`.

In general I would say that Aurora tries to be as compatible as possible, but there's always a risk that some detail works slightly different.
I would assume that Amazon has reimplemented major parts of the Postgres codebase.

You could also try to run Dex' automated tests against your target Aurora database.
Currently they're run against Postgres 10.8: https://github.com/dexidp/dex/blob/master/.github/workflows/ci.yaml

**Reply by AnkitaBarbi** [reply]
Thanks @heidemn . Just wanted to let inform that I tried to connect it against aurora serverless Postgres and was able to achieve it. so far it looks good. lets see how it goes further
